src/components/Header.js

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import bell from '../assets/imgs/notification-bell.svg';
+import settingsIcon from '../assets/imgs/settings-icon.svg';
+import kobe from '../assets/kobe.jpg';
+import raptorsLogo2 from '../assets/imgs/raptors-logo2.svg';
 
 const Header = () => {
     return (
-        <ul className="nav-bar">
-            <li>Raptors logo</li>
-            <Link to='/'>Players</Link>
-            <Link to='/standings'>Standings</Link>
-            <Link to='/games'>Games</Link>
-            <li>username</li>
-        </ul>
+        <div className="header-container">
+            <div className="team">
+                <div className="logo-container">
+                    <img src={raptorsLogo2} title="raptors logo" alt="raptors logo"></img>
```

Comment:
One more thing. Let's remove the raptorsLogo2 and raptors.svg file and instead use this link for the src:

https://cdn.nba.net/assets/logos/teams/secondary/web/TOR.svg

Also the issue we are seeing is because the svg viewport is messing up the circular logo. In the near future we'll make it look cleaner, for now let's just use the online svg link instead.